app/web/features/communities/CommunitiesPage/CommunitiesPage.tsx

```diff
@@ -0,0 +1,280 @@
+import {
+  Divider,
+  List,
+  ListItem,
+  ListItemText,
+  Typography,
+} from "@mui/material";
+import MuiLink from "@mui/material/Link";
+import Alert from "components/Alert";
+import Button from "components/Button";
+import CenteredSpinner from "components/CenteredSpinner/CenteredSpinner";
+import PageTitle from "components/PageTitle";
+import StyledLink from "components/StyledLink";
+import {
+  useCommunity,
+  useListSubCommunities,
+} from "features/communities/hooks";
+import { Trans, useTranslation } from "i18n";
+import { COMMUNITIES, DASHBOARD, GLOBAL } from "i18n/namespaces";
+import { Community } from "proto/communities_pb";
+import React, { useEffect, useRef, useState } from "react";
+import { routeToCommunity } from "routes";
+import makeStyles from "utils/makeStyles";
+
+const COMMUNITY_BUILDER_FORM_LINK =
+  "https://couchers.org/community-builder-form";
+const COMMUNITY_GUIDELINES_LINK =
+  "https://docs.google.com/document/d/1A9vCdKGQ_WKoyjCh5KScdGgpnighr9yL_F0-OaPT2yA/edit?usp=sharing";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexDirection: "row",
+    alignItems: "flex-start",
+    "& > * + *": {
+      marginInlineStart: theme.spacing(2),
+    },
+  },
+  list: {
+    minWidth: "10rem",
+  },
+  loader: {
+    margin: theme.spacing("auto", 2),
+  },
+  selected: {
+    fontWeight: "bold",
+  },
+  emptyState: {
+    color: theme.palette.grey[600],
+  },
+  createCommunityText: {
+    paddingBlockStart: theme.spacing(2),
+    paddingBlockEnd: theme.spacing(2),
+  },
+  headerRow: {
+    display: "flex",
+    justifyContent: "space-between",
+    width: "100%",
+    paddingBottom: theme.spacing(1),
+  },
+}));
+
+const CommunitiesPage = () => {
+  const { t } = useTranslation([COMMUNITIES, DASHBOARD, GLOBAL]);
+  const classes = useStyles();
+  const [selected, setSelected] = useState<Community.AsObject[]>([]);
+
+  const globalCommunityQuery = useCommunity(1);
+
+  //react-query doesn't have useInfiniteQueries
+  //as a workaround, cache query results
+  //and only show "Load more" for last column
+  const query = useListSubCommunities(
+    selected?.[selected.length - 1]?.communityId || 1,
+  );
+  const [cachedQueryResults, setCachedQueryResults] = useState<
+    {
+      data: Community.AsObject[];
+      hasMore?: boolean;
+    }[]
+  >([]);
+
+  const lastColumnRef = useRef<HTMLDivElement>(null);
+  useEffect(() => {
+    setTimeout(
+      () =>
+        lastColumnRef.current?.scrollIntoView({
+          behavior: "smooth",
+          inline: "end",
+        }),
+      50,
+    );
+  }, [selected]);
```

Comment:
When I first load the Communities page it scrolls me down below the heading which is a bit confusing. I think because of this?